Resources/Prototypes/Traits/speech.yml

```diff
@@ -64,14 +64,12 @@
     - type: ReplacementAccent
       accent: liar
 
-# 2 Cost
-
 - type: trait
   id: SocialAnxiety
   name: trait-socialanxiety-name
   description: trait-socialanxiety-desc
   category: SpeechTraits
-  cost: 2
+  cost: 1
```

Comment:
This is unrelated to your PR, and should be a commented change

Reply:
It was related due to me wanting more variety, but comments have been added-